### PR TITLE
add: styles and structure

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

File: src/frontend/ProductFilter/ProductFilter.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.product-container {
+  margin-top: 2rem;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(310px, 1fr));
```
Comment (Chakravarthi0) [inline src/frontend/ProductFilter/ProductFilter.css:4]: you can use rem unit here

File: src/frontend/ProductFilter/ProductFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+              <input name="rating" type="radio" />4 Star and above
+            </label>
+            <label for="rating">
+              <input name="rating" type="radio" />3 Star and above
+            </label>
+            <label for="rating">
+              <input name="rating" type="radio" />
+              Star and above
+            </label>
+          </filter-section>
```
Comment (Chakravarthi0) [inline src/frontend/ProductFilter/ProductFilter.jsx:52]: you forgot to add filter by : "2 stars and above " & "1 star and above"

File: src/frontend/ProductFilter/ProductFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+          <filter-section className="flex-column">
+            <h2>Rating</h2>
```
Comment (Chakravarthi0) [inline src/frontend/ProductFilter/ProductFilter.jsx:55]: you can name this as "sort by"

File: src/frontend/ProductFilter/ProductFilter.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            <button>clear</button>
+          </filter-section>
+
+          <filter-section>
+            <h2>Price</h2>
+            <div className="price-range flex-row">
+              <span>50</span>
+              <span>150</span>
+              <span>200</span>
+            </div>
```
Comment (Chakravarthi0) [inline src/frontend/ProductFilter/ProductFilter.jsx:20]: suggestion: you can add some gap between the elements.

image

**Review by Chakravarthi0 — APPROVED** [review]
apart from my suggestions, everthing looks good

**Review by percius47 — APPROVED** [review]
styles PR, good use of functions like repeat and minmax, clean folder structure as well.

**Review by KoushikKintali — COMMENTED** [review]
Looks good, please implement the changes suggested by @Chakravarthi0

**Pull request merged**